README.md

```diff
@@ -40,11 +40,12 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 
```

Comment:
- [ ] Please add information to run your project locally to set it up and much more that can make your README very professional